.ci/build-n-run.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+uname -r
+cd examples
+make
+cd ..
+exclude_list=$(cat .ci/NotTest.txt);   #the module in NotTest.txt will be ignored by the shell
```

Comment:
`NotTest` is ambiguous. It is hard to distinguish from "not tested" and "non-working." In this case, you meant "non-working" modules which should be excluded.